backend/src/main/kotlin/org/loculus/backend/service/files/FilesDatabaseService.kt

```diff
@@ -49,4 +47,21 @@ class FilesDatabaseService(private val dateProvider: DateProvider) {
         .map { it[FilesTable.releasedAtColumn] }
         .first()
         .let { it != null }
+
+    /**
+     * Return the subset of file IDs that either do not exist or have not been checked yet.
```

Comment:
Is the comment accurate? The function doesn't return file IDs that do not exist, but only those for which the size is not known, right?